### PR TITLE
Volatile chat endpoints

### PR DESCRIPTION
**Pull request by eshaibu**
### What does this PR do?
Create a socket for volatile chat using Syncano realtime feature.

### CODE REVIEW

File: socket.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: volatile-chat
+description: Volatile chat socket using using Syncano realtime feature
```
Comment (adamwardecki) [inline socket.yml]: double 'using'

File: socket.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        exit_code: 400
+        description: Failed
+        examples:
+          - |
+            {
+              "message": "Failed to publish message"
+            }
+  group-chat:
+    channel: group.{room}
+    description: Endpoint to subcribe to poll for a group messages. Example group.group_sports
```
Comment (adamwardecki) [inline socket.yml]: example should be a yml property, not a part of the description

**Review by adamwardecki — CHANGES_REQUESTED** [review]
Also:
- use Syncano Validate lib for request validation.
- @syncano/cli should be listed as a dependency

**Comment by eshaibu** [comment]
The Syncano Validate lib validates if a request contains a required parameter but does not check if the parameter is empty. I have a function `checkRequired` in `src/utils/helpers` checks for both required parameter and if it's value is empty. Since it does this, adding Syncano validate library will be adding a lib to package json which is not being utilised.

**Review by adamwardecki — APPROVED** [review]

**Pull request merged**